backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/rating/BpkRating.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.rating
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import net.skyscanner.backpack.compose.rating.internal.BpkRatingLayout
+import net.skyscanner.backpack.compose.text.BpkText
+
+enum class BpkRatingSize {
+  Base,
+  Large,
+}
+
+enum class BpkRatingScale {
+  ZeroToFive,
+  ZeroToTen,
+}
+
+@Composable
+fun BpkRating(
+  title: String,
+  value: Float,
+  modifier: Modifier = Modifier,
+  scale: BpkRatingScale? = BpkRatingScale.ZeroToFive,
+  size: BpkRatingSize = BpkRatingSize.Base,
+  subtitle: String? = null,
+) {
+  BpkRatingLayout(
+    value = value,
+    scale = scale,
+    size = size,
+    subtitle = subtitle,
+    modifier = modifier
+  ) {
+    BpkText(
+      text = title,
+      maxLines = 1,
+      overflow = TextOverflow.Ellipsis,
+    )
+  }
+}
+
+@Composable
+fun BpkRating(
+  value: Float,
+  modifier: Modifier = Modifier,
+  scale: BpkRatingScale? = BpkRatingScale.ZeroToFive,
```

Comment:
Showing the scale is different from the supported scale. For example the trip advisor example doesn't show the scale, but the logo still has a 0-5 scale, therefore we should clamp it to that. So the `showScale` property still makes sense and making this non-nullable.